Sync/Synchronizers/Bookmarks/BookmarksSynchronizer.swift

```diff
@@ -118,34 +118,37 @@ open class BufferingBookmarksSynchronizer: TimestampedSingleCollectionSynchroniz
 
         statsSession.start()
         if !AppConstants.shouldMergeBookmarks {
-            if case .release = AppConstants.BuildChannel {
-                // On release, just mirror; don't validate.
-                run = doMirror
-            } else {
-                run = doMirror >>== effect({ result in
-                    // Just validate to report statistics.
-                    if case .completed = result {
-                        log.debug("Validating completed buffer download.")
-                        let _ = buffer.validate()
+            run = doMirror >>== { result in
```

Comment:
@rnewman Just to double check - are we okay with running buffer validation on all channels when merging is off so we can collect telemetry data from the sync ping?

My end goal is to release this in v8.x to start collecting a baseline of data prior to the bookmark repair work.